### PR TITLE
Fix issue #1274: Add QuerySeter.Distinct()

### PR DESCRIPTION
**Pull request by pjoe**
Sample use:
```go
var permissions []*models.Permission
_, err = o.QueryTable("policy").
	Filter("Groups__Group__Users__User", user).
	Distinct().
	All(&permissions)
```

### CODE REVIEW

File: orm/orm_queryset.go [hunk 1]
```diff
@@ -112,24 +113,30 @@ func (o querySet) OrderBy(exprs ...string) QuerySeter {
...
+			case string:
+				o.related = append(o.related, val)
+			case int:
+				o.relDepth = val
+			default:
+				panic(fmt.Errorf("<QuerySeter.RelatedSel> wrong param kind: %v", val))
+			}
+		}
+	}
+	return &o
```
Comment (astaxie) [inline orm/orm_queryset.go]: please use the space, not tab

**Comment by pjoe** [comment]
Weird, I didn't actually touch that part of the file, but think I have go_fmt on save in my editor. Anyway will update with tab/spaces fixed

**Comment by pjoe** [comment]
Hmmm looking through the file (and other files) that functions seems to be the only one using space, everywhere else is tabs? So i guess my go_fmt decided to try and fix that. Anyway can try and remove that part from the PR

**Comment by astaxie** [comment]
LGTM

**Pull request merged**